user-guide/Troubleshooting/Known_issues/KI_Upgrade_fails_VerifyNoObsoleteAPIDeployed_prerequisite.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: KI_Upgrade_fails_VerifyNoObsoleteApiDeployed_prerequisite
+---
+
+# Upgrade fails because of VerifyNoObsoleteApiDeployed.dll prerequisite
+
+## Affected versions
+
+From DataMiner 10.4.0 onwards
+
+## Cause
+
+From DataMiner 10.4.0 onwards, the obsolete *API Deployment* feature will be completely removed from DataMiner and replaced by [User-Defined APIs](xref:UD_APIs). The VerifyNoObsoleteApiDeployed upgrade prerequisite verifies no obsolete APIs are deployed before upgrading as these won't work anymore after the upgrade.
+
+## Fix
+
+If you don't want to lose your APIs, you will have to migrate them to use the new User-Defined APIs feature. There is documentation on how to [define a new API](xref:UD_APIs_Define_New_API) and how to [create an API from an existing automation script](xref:UD_APIs_Using_existing_scripts).
+
+If you just want to remove your obsolete APIs from your system, follow below steps. Note that you'll also have to follow these steps after you migrated your APIs to the *User-Defined APIs* feature.
+
+1. Open DataMiner Cube, and log into your DataMiner System.
+
+1. Go to *System Center* > *API deployment (Deprecated)*
+
+1. Your API's will be displayed in a tree view
```

Comment:
```suggestion
1. Your APIs will be displayed in a tree view
```

Reply:
Also indent this since this is not action but a result of what you will see after the previous action.